asset-transfer-private-data/chaincode-go/privateAssetTransfer.go

```diff
@@ -0,0 +1,469 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+)
+
+// Asset describes main asset details that are visible to all organizations
+type Asset struct {
+	ObjectType 		 string `json:"object_type"`  //docType is used to distinguish the various types of objects in state database
+	ID             string `json:"asset_id"`
+	Color          string `json:"color"`
+	Size           int    `json:"size"`
+	Owner          string `json:"owner"`
+}
+
+// AssetPrivateDetails describes details that are private to owners
+type AssetPrivateDetails struct {
+	ID             string `json:"asset_id"`
+	AppraisedValue int    `json:"appraisedValue"`
```

Comment:
I guess we are using `snake_case` convention for all json fields. so rename the json field 
```suggestion
	AppraisedValue int    `json:"appraised_value"`
```